nix/cells/cloud/actions.nix

```diff
@@ -15,8 +15,8 @@
 
       #lib.merge
       #ios: [
-        #lib.io.github_push & github,
-        { #lib.io.github_pr, github, #target_default: false },
+        {#lib.io.github_push, github, #default_branch: true},
```

Comment:
We also need to build on Bors-related branches:

> Your CI system should build the “staging” and “trying” branches

https://bors.tech/documentation/getting-started/

See e.g. #4372 where Cicero did not run on the Bors commits.

Reply:
Ah thanks, fixed :slightly_smiling_face: